samples/deployment/test-network/setup.sh

```diff
@@ -12,81 +12,67 @@ if [[ -z "${FPC_PATH}" ]]; then
   exit 1
 fi
 
-backup() {
-  FILE=$1
-  BACKUP="${FILE}.backup"
-  echo "backup ${FILE} ..."
-  if [[ -e "${BACKUP}" ]]; then
-    cp "${BACKUP}" "${FILE}"
-  else
-    cp "${FILE}" "${BACKUP}"
-  fi
-}
+
+##############################################################################################
+# Get the fabric samples repo and fetch binaries and container images
+##############################################################################################
 
 echo "Prepare fabric samples test-network for FPC"
 
 FABRIC_SAMPLES=${FPC_PATH}/samples/deployment/test-network/fabric-samples
-CORE_PATH=${FABRIC_SAMPLES}/config/core.yaml
-DOCKER_PATH=${FABRIC_SAMPLES}/test-network/docker
-DOCKER_COMPOSE_TEST_NET=${DOCKER_PATH}/docker-compose-test-net.yaml
-DOCKER_COMPOSE_CA=${DOCKER_PATH}/docker-compose-ca.yaml
 
-if [ ! -d "${FABRIC_SAMPLES}/bin" ]; then
-  echo "Error: no fabric binaries found, see README.md"
-  exit 1
+if [ ! -d "${FABRIC_SAMPLES}" ]; then
+  echo "Fabric samples not found! cloning now!"
+  git clone https://github.com/hyperledger/fabric-samples "${FABRIC_SAMPLES}"
+  pushd "${FABRIC_SAMPLES}"
+  git checkout -b "works" 50b69f6
+  popd
```

Comment:
So now there is a download in a temp folder, and a clone in what was the original module folder.
Any reason for this added complexity?

Reply:
@bvavala what temp directory? this was always like this, but before we had the user manually execute these instructions, now we have just automated it for them .... the git clone and git checkout were in the Readme before.

Reply:
See `deployment-test.sh`.
I see that the commands were in the repo, but now I'm wondering:
- why do we have a git clone in a folder where the same repo is already supposed to be initialized as a module?
- also, we have the module initialized at commit X, but we clone and checkout commit Y. Why so?

Reply:
I have mixed feelings about having fabric-samples as git submodule. When working with the test-network is often happens that you want to wipe out the entire folder and start from scratch. Unfortunately, their test-network does not have a neat `make clean` to do that job. Well one could argue one could just use git to reset. I don't see any issue with a quick git clone. But maybe keeping fabric samples a submodule would be better. mhhhhh

Reply:
@bvavala @mbrandenburger but, for this PR we are just upgrading from what we currently had. I do think we can think about changing it, but I would do that in another PR ... Remember the objective for this PR is to upgrade the fabric network used in the test-network samples because it broke when we upgraded the docker version used in the project.